### PR TITLE
Add arm32 Ubuntu distributions as supported

### PR DESCRIPTION
**Pull request by sxa**
At the moment the supported platforms pages have nothing listed for arm32/armv7l/aarch32

Signed-off-by: Stewart Addison <sxa@uk.ibm.com>

##### Checklist

- [ ] `npm test` passes
- [ ] documentation is changed or added (if applicable)
- [ ] permission has been obtained to add new logo (if applicable)
- [ ] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/handlebars/supported_platforms.handlebars [hunk 1]
```diff
@@ -45,6 +45,7 @@
                     <th>x64</th>
                     <th>ppc64le</th>
                     <th>s390x</th>
+                    <th>aarch32</th>
```
Comment (smlambert) [inline src/handlebars/supported_platforms.handlebars:48]: Does the API serve up as aarch32 or as arm32?  I have seen both ways of naming this platform... we should likely attempt to consolidate on 1 way of naming...

**Review by smlambert — APPROVED** [review]
Other than the question around naming aarch32 versus arm32, the changes LGTM.  Will leave it up to @sxa to merge (depending on what you decide to do with the name).

**Review by gdams — APPROVED** [review]

File: src/handlebars/supported_platforms.handlebars (hunk 1 above)
Comment (sxa) [reply src/handlebars/supported_platforms.handlebars:48]: Neither ... The API uses `arm` at the moment for 32-bit ARM but I agree this was the one thing I was undecided about, but chose to make it the same as the other columns that were in the file for now.

**Comment by sxa** [comment]
General approval that this is a tactical fix and we should look at merging naming outside of this PR

**Pull request merged**